proctor-codegen/build.gradle

```diff
@@ -0,0 +1,28 @@
+indeedOss.activateFeature 'library'
+indeedLibrary.name = 'proctor-codegen'
+
+dependencies {
+    implementation project(':proctor-common')
+    implementation 'org.freemarker:freemarker:2.3.16'
+    implementation libs.guava
+    implementation libs.commonsLang
+    implementation libs.jacksonDatabind
+    testImplementation libs.annotationApi
+    testImplementation libs.jsr305
+    testImplementation libs.junit
+    testImplementation libs.assertj
+}
+
+tasks.named("test").configure { dependsOn(":proctor-common:shadowJar") }
```

Comment:
This shouldn't be needed in any of the build.gradles.

Reply:
Get the following error without it:

``` A problem was found with the configuration of task ':proctor-codegen:test' (type 'Test').
  - Gradle detected a problem with the following location: '/Users/zgoodwin/opensource/fork/proctor/proctor-common/build/libs/proctor-common.jar'.
    
    Reason: Task ':proctor-codegen:test' uses this output of task ':proctor-common:shadowJar' without declaring an explicit or implicit dependency. This can lead to incorrect results being produced, depending on what order the tasks are executed.
    
    Possible solutions:
      1. Declare task ':proctor-common:shadowJar' as an input of ':proctor-codegen:test'.
      2. Declare an explicit dependency on ':proctor-common:shadowJar' from ':proctor-codegen:test' using Task#dependsOn.
      3. Declare an explicit dependency on ':proctor-common:shadowJar' from ':proctor-codegen:test' using Task#mustRunAfter.
    
    Please refer to https://docs.gradle.org/8.1.1/userguide/validation_problems.html#implicit_dependency for more details about this problem.```

Reply:
REmoved